build/params_calibnet.go

```diff
@@ -73,11 +73,11 @@ const UpgradeSharkHeight = 16800 // 6 days after genesis
 // 2023-02-21T16:30:00Z
 const UpgradeHyggeHeight = 322354
 
-// 2023-04-20T16:30:00Z
-const UpgradeLightningHeight = 489394
+// 2023-04-20T14:00:00Z
```

Comment:
I believe this is off by a half hour, right?

Reply:
Or the epoch is off. The difference between 322354 and 489154 is 166800, or 1390 hours, but we're losing a half-hour here.

Reply:
thx, this should be `489094`